### PR TITLE
Change in README.md file.

### PR DESCRIPTION
**Pull request by Kartik-Khandelwal**
# Change in 'Follow these simple steps:' in the README.md file.
### -> Bold.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Open-source maintainers are always looking to get more people involved, but new
 
 You're welcome to add a new project in Good First Issue, and we encourage all projects &mdash; old and new, big and small.
 
-Follow these simple steps:
+### <b>Follow these simple steps:</b>
```
Comment (sanket-deepsource) [inline README.md:21]: Please remove the `###` declaration, since this will convert the line into an `<h3>`.

Comment (Kartik-Khandelwal) [reply README.md:21]: Sure sir, I will remove "###".

Comment (Kartik-Khandelwal) [reply README.md]: This is my first contribution and as per your suggestion i had removed "###".

**Comment by Kartik-Khandelwal** [comment]
Sir, how can I revert changes in the file. Please tell me as I am new in this field.

**Comment by jai-deepsource** [comment]
Closing this due to inactivity. Please re-open if necessary.

@sanket-deepsource

**Pull request closed without merging**